index.html

```diff
@@ -229,30 +237,102 @@ <h2>Valles Caldera</h2>
+
+                        <h3><a href="https://journals.sagepub.com/doi/full/10.1177/1475921720977017">Enabling human–infrastructure interfaces for inspection using augmented reality</a></h3>
+                        <i>Maharjan, D and Agüero, M and Mascarenas, D and Fierro, R and Moreu, F.</i>
+                        <p>This article studies the role of Augmented Reality (AR) technology as a tool to increase human
+                            awareness of infrastructure in their inspection work. The domains of interest of this research
+                            include both infrastructure inspections (emphasis on the collection of data of structures to
+                            inform management decisions) and emergency management (focus on the data collection of the
+                            environment to inform human actions)...</p>
+
+                        <h3><a href="https://ieeexplore.ieee.org/abstract/document/10018733">Multi-Robot system for autonomous cooperative counter-UAS missions: Design, integration, and field testing</a></h3>
+                        <i>A. Barisic, M. Ball, N. Jackson, R. McCarthy, N. Naimi, L. Strassle, J. Becker, M. Brunner, J. Fricke, L. Markovic, I. Seslar, D. Novick, J. Salton, R. Siegwart, S. Bogdan, and R. Fierro</i>
+                        <p>With the rapid development of technology and the proliferation of uncrewed aerial systems (UAS),
+                            there is an immediate need for security solutions. Toward this end, we propose the use of a
+                            multi-robot system for autonomous and cooperative counter-UAS missions. In this paper, we present
+                            the design of the hardware and software components of different complementary robotic
+                            platforms: a mobile uncrewed ground vehicle (UGV) equipped with a LiDAR sensor, an uncrewed aerial vehicle (UAV)
+                            with a gimbal-mounted stereo camera for air-to-air inspections, and a UAV with a capture mechanism equipped with radars and camera...</p>
+
+                        <h3><a href="https://ieeexplore.ieee.org/abstract/document/10025390">Robust Linear-Velocity-Free Formation Tracking of Multiple Quadrotors with Unknown Disturbances</a></h3>
+                        <i>J. Lin, Y. Wang, Z. Miao, Q. Lin, G. Hu, and R. Fierro</i>
+                        <p>This article addresses the problem of quadrotor formation control with inaccessible linear
+                            velocity and the unknown disturbances under directed interaction topologies. Considering the
+                            unknown disturbances act in both the translational and rotational motions of each underactuated
+                            quadrotor, a hierarchical distributed disturbance rejection controller is developed for the
+                            quadrotor formation...</p>

```

Comment:
@johncarl81 What is the purpose of the `...` here?

Reply:
The abstracts are huge.  I abbreviated them.

Reply:
cool so do we want to keep the `...`s?

Reply:
Sorry If you did not get the notification but @johncarl81 do we want to keep the `...`s? if so then looks good and I'll merge and do a pull on the cs systems

Reply:
Yeah, lets keep them.  We actually had them [previously](https://github.com/BCLab-UNM/volcan_website/blob/master/index.html#L239).